backend/src/services/identity-project/identity-project-dal.ts

```diff
@@ -208,9 +236,36 @@ export const identityProjectDALFactory = (db: TDbClient) => {
     }
   };
 
+  const getCountByProjectId = async (
```

Comment:
You can do it two ways. Either this query can just use the SQL count operator rather than taking length or you can make the above query actual list query support count using a DENSE_RANK like we do for secret approval to get total count in one single db call